doc/source/reference/commands/select.rst

```diff
@@ -2453,6 +2455,55 @@ specify one key as ``drilldowns[${LABEL}].keys`` like ``--drilldowns[tag].keys
 tag``. You should use ``_key`` for the case. It's the same rule in
 :ref:`select-drilldown-output-columns`.
 
+.. _select-drilldowns-label-table:
+
+``drilldowns[${LABEL}].table``
+""""""""""""""""""""""""""""""
+
+.. versionadded:: 6.0.2
+
+Specifies ``${LABLE}`` of other ``drilldown``, ``drilldowns`` or ``slices``.
+
+The result of the specified ``${LABLE}`` is drilldowned,
+which means that this paramter enables nested aggregate calculations and groups in drilldown.
+
+Here is an example to execute nested drilldown by tag and then by category.
+
+.. groonga-command
+.. include:: ../../example/reference/commands/select/drilldowns_label_table.log
+.. table_create Tags TABLE_PAT_KEY ShortText
+.. column_create Tags category COLUMN_SCALAR ShortText
+.. table_create Memos TABLE_HASH_KEY ShortText
+.. column_create Memos tag COLUMN_SCALAR Tags
+.. load --table Memos
+.. [
+.. {"_key": "Groonga is fast!", "tag": "Groonga"},
+.. {"_key": "Groonga sticker!", "tag": "Groonga"},
+.. {"_key": "Mroonga sticker!", "tag": "Mroonga"},
+.. {"_key": "Rroonga is fast!", "tag": "Rroonga"}
+.. ]
+.. load --table Tags
+.. [
+.. {"_key": "Groonga", "category": "C/C++"},
+.. {"_key": "Mroonga", "category": "C/C++"},
+.. {"_key": "PGroonga", "category": "C/C++"},
+.. {"_key": "Rroonga", "category": "Ruby"}
+.. ]
+.. select Memos \
+..   --drilldowns[label_tag].keys tag \
+..   --drilldowns[label_tag].output_columns _key \
+..   --drilldowns[label_category].table label_tag \
+..   --drilldowns[label_category].keys category \
+..   --drilldowns[label_category].output_columns _key,_nsubrecs
+
+In this example, the schema contains the table named as ``Memo`` which has the column named as ``tag`` 
```

Comment:
```suggestion
In this example; 
The schema contains the table named as ``Memo`` which has the column named as ``tag``.
the table named as ``Tags`` which has the columns named as ``category``.
```
when it comes to line up fact, breaking up sentence would be easier to understand.